user-guide/Advanced_Modules/DOM/DOM_objects/DOM_FieldDescriptors/DOM_ResourceFieldDescriptor.md

```diff
@@ -0,0 +1,29 @@
+---
+uid: DOM_ResourceFieldDescriptor
+---
+
+# ResourceFieldDescriptor
+
+- **FieldValue type**: `GUID` of an SRM Resource
+- **Multiple values supported**: :heavy_check_mark:
+
+| Type of Descriptor | FieldType | FieldValue type |
+|--------------------|-----------|-----------------|
+| Resource | Guid | Guid |
+| Resource with multiple values enabled| List\<Guid\> | Guid (ListValueWrapper) |
+
+Defines a field that has the `GUID` of an SRM `Resource`.
+
+The `ResourceFieldDescriptor` lists all `Resources` in the defined `ResourcePools`, the selected `Resource` is saved as their `GUID`.
```

Comment:
Is it mandatory to fill in one or multiple pools? Can you leave the list empty to show all resources?